src/Uno.Core/Disposables/IExtensibleDisposable.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Uno.Disposables
+{
+	/// <summary>
+	/// Represents a disposable object on which some external disposables can be linked to, in order to share it lifetime.
+	/// <remarks>
+	/// Default implementation:<br />
+	/// <br />
+	/// private readonly CompositeDisposable _subscriptions = new CompositeDisposable();<br />
+	/// public IDisposable LinkDisposable(IDisposable disposable) => _subscriptions.Register(disposable);<br />
+	/// public void Dispose() => _subscriptions.Dispose();
+	/// </remarks>
+	/// </summary>
+	public interface IExtensibleDisposable : IDisposable
+	{
+		/// <summary>
+		/// Currently registered extensions on the registry
+		/// </summary>
+		/// <remarks>
+		/// To get a known service, use Extensions.OfType&lt;TService&gt;()
```

Comment:
Service ?

Reply:
"TService" because this interface is designed to attach a 'Service' / 'Behavior' to an object.

Reply:
Ok I renamed it to TExtension to be more consistent